2008-01-12  H.J. Lu  <hongjiu.lu@intel.com>

	* sysdeps/i386/fpu/ftestexcept.c (fetestexcept): Use short for fnstsw.

Index: glibc-2.5-20061008T1257/sysdeps/i386/fpu/ftestexcept.c
===================================================================
--- glibc-2.5-20061008T1257.orig/sysdeps/i386/fpu/ftestexcept.c
+++ glibc-2.5-20061008T1257/sysdeps/i386/fpu/ftestexcept.c
@@ -26,7 +26,7 @@
 int
 fetestexcept (int excepts)
 {
-  int temp;
+  short temp;
   int xtemp = 0;
 
   /* Get current exceptions.  */
